coffeelint.json

```diff
@@ -14,11 +14,24 @@
   "no_debugger": {
     "level": "error"
   },
-  "indentation": {
-    "value": 2,
```

Comment:
Just for posterity, this is enforced by coffeelint by default, so we don't need to specify it.